content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -160,7 +160,12 @@ for more information.
 
 This section contains the necessary steps to install `CRI-O` as CRI runtime.
 
-Use the following commands to install CRI-O on your system (but be sure the major/minor version of CRI-O matches the major/minor version of kubernetes (see the [cri-o compatiblity matrix](https://github.com/cri-o/cri-o)):
+Use the following commands to install CRI-O on your system:
+
+{{< note >}}
```

Comment:
@larntz-tbc , Changes look good. I would have placed the note before Line 163, but seems fine as is.
/lgtm